examples/snippets/modules/ROOT/examples/build-akka-streamlets-java/step0/src/main/java/com/example/ReportPrinter.java

```diff
@@ -0,0 +1,18 @@
+package com.example;
+
+import akka.NotUsed;
+import akka.stream.*;
+import akka.stream.javadsl.*;
+
+import cloudflow.streamlets.*;
+import cloudflow.streamlets.avro.*;
+import cloudflow.akkastream.*;
+import cloudflow.akkastream.javadsl.*;
+
+public class ReportPrinter extends AkkaStreamlet {
+  // 1. TODO Create inlets and outlets
+  // 2. TODO Define the shape of the streamlet
+  public StreamletShape shape() { throw new UnsupportedOperationException("Not Implemented"); }
+  // 3. TODO Override createLogic to provide StreamletLogic
+  public RunnableGraphStreamletLogic createLogic() { throw new UnsupportedOperationException("Not Implemented"); }
+}
```

Comment:
!WHY LUC WHY!
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
lol I'm just kidding of course.